lib/Semantics/resolve-names.cpp

```diff
@@ -4649,7 +4675,7 @@ std::optional<DerivedTypeSpec> DeclarationVisitor::ResolveDerivedType(
       details.set_isForwardReferenced();
       symbol->set_details(std::move(details));
     } else { // C883
-      Say(name, "Derived type '%s' not found"_err_en_US);
+      Say(name, "Derived type '%s' not found"_err_en_US); // C732
```

Comment:
If we're truly checking both C732 and C883 correctly here, combine the comments.

Reply:
Good point.  I'll combine them.

Reply:
Actually, I don't think that this check has anything to do with constraint C883.  I'll remove the reference.